pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -113,6 +113,15 @@ type PrebindPlugin interface {
 	Prebind(pc *PluginContext, p *v1.Pod, nodeName string) *Status
 }
 
+// PostbindPlugin is an interface that must be implemented by "postbind" plugins.
+// These plugins are called after a pod being scheduled
+type PostbindPlugin interface {
+	Plugin
+	// Postbind is called after binding a pod. All postbind plugins must return
+	// success or will can not clean up state associated with the reserved pod.
```

Comment:
can you please rephrase the last part of the comment, specifically the part "or will can not clean up state associated with the reserved pod", it is not clear to me what happens on failure.

Reply:
This is a description from the design documentation: [post-bind](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20180409-scheduling-framework.md#post-bind)

Reply:
I guess my point is that the sentence has typos, how about we use the exact same text from the description:

"Post-bind plugins are called after a pod is successfully bound. This is the end of a binding cycle, and can be used to clean up associated resources."

Reply:
or perhaps minor modifications to your comment:

"Postbind is called after binding a pod. All postbind plugins must return success, otherwise state associated with the reserved pod will not be cleaned up."

Reply:
DONE,thanks